lib/explorer.js

```diff
@@ -20,16 +20,21 @@ function verifyCoinbaseSupply () {
 }
 
 function coinbase_supply () {
-  return promisify(
-    Tx.find.bind(Tx),
-    { 'vin': { $elemMatch: { 'addresses': 'coinbase', 'amount': { $gt: 0 } } } }
-  ).then(([ err, txes ]) =>
-    txes.reduce((sum, tx) => sum + tx.vin.find(vin => vin.addresses === 'coinbase').amount, 0)
-  )
+  return requestp(base_url + 'gettxoutsetinfo').then(info => info.total_amount)
+  // return promisify(
+  //   Tx.find.bind(Tx),
+  //   { 'vin': { $elemMatch: { 'addresses': 'coinbase', 'amount': { $gt: 0 } } } }
+  // ).then(([ err, txes ]) =>
+  //   txes.reduce((sum, tx) => sum + tx.vin.find(vin => vin.addresses === 'coinbase').amount, 0)
+  // )
```

Comment:
Remove deprecated?

Reply:
yes, there are a couple legacy ways the explorer was calculating total supply that I'll remove now that I've found that `gettxoutsetinfo` returns that info anyways.